packages/patternfly-4/react-core/src/components/Alert/Alert.tsx

```diff
@@ -70,7 +70,7 @@ const Alert: React.FunctionComponent<AlertProps & InjectedOuiaProps> = ({
       className={customClassName}
       aria-label={ariaLabel}
       {...(ouiaContext.isOuia && {
-        'data-ouia-component-type': 'Alert',
+        'data-ouia-component-type': 'PF4/Alert',
```

Comment:
Probably want an issue for this to discuss the format. For example, we may want to omit the version and use something more like `pf-alert`?

Reply:
I agree with @dlabrecq . Is there a formal spec for this?

Reply:
@dlabrecq What if a someday new version of Patternflywill be released let's say PF5 and some application will have to use both of them?

Reply:
@redallen There is a spec https://ouia.readthedocs.io/en/latest/README.html#ouia-component:

```
A root level HTML element with a data-ouia-component-type attribute describing a unique name identifying ALL HTML components that can be controlled with the same code or interactions. These identifiers MAY and SHOULD be namespaced when used within a framework, particularly when the name is generic. The delimiter between namespace and type name should be a single / character.

    e.g. A page that has a special dropdown could choose to name that dropdown as FrameworkA/CustomDropdown. All instances of this FrameworkA/CustomDropdown component MUST be expected to be able to be controlled via the same automation.
```

Reply:
We don't have this for our CSS selector names, so thought to ask. I'm fine with whatever the team decides here. Just thought the team should discuss...